corona-warn-app/cwa-app-android
# Reissuance badge (EXPOSUREAPP-11936,11937)

## Changes
- Dcc Reissuance tile 
- Show / dismiss dcc Reissuance badge when user open dcc reissuance  consent screen
- Show notification
- Open person details when notification is clicked 
- Tile screenshots
- Some refactoring for booster badge

## How to test?
- Against Mock-Cloud
- Scan certificates from `cwa dcc series -e cf --preset 6 -o` presets (3,4,5,6) should work
- Click notification to open the details screen
- Open consent screen 
- Notice that badge change is reflected everywhere 

## Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11936
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11937
